anime_downloader/extractors/waaw.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+# -*- coding:Utf-8 -*-
+#waaw.py
+import re
+import json
+import sys
+from anime_downloader.extractors.base_extractor import BaseExtractor
+from anime_downloader.sites import helpers
+import logging
+
+logger = logging.getLogger(__name__)
+
+from anime_downloader import util
+import os
+import tempfile
+
+
+
+chromedriver = util.check_in_path('chromedriver')
```

Comment:
Because it was not installed by default. you need to install it to selinium work fine:
# brew cask install chromedriver
# sudo apt install chromium-chromedriver

Reply:
If those checks aren't done or are lacking changes should be made in [selescrape](https://github.com/vn-ki/anime-downloader/blob/master/anime_downloader/sites/helpers/selescrape.py) instead. Providers/extractors shouldn't need to worry about that

Reply:
yes right, but helpers don't make it. but it's a code line not used.. that thing just print a warning to the other users ... not for me, or for me, just to remember me that i have to install something